examples/pvtol-nested.py

```diff
@@ -37,110 +39,118 @@
 #
 
 # Design a simple lead controller for the system
-k = 200;  a = 2;  b = 50;
-Ci = k*tf([1, a], [1, b]);		# lead compensator
-Li = Pi*Ci;
+k, a, b = 200, 2, 50
+Ci = k*tf([1, a], [1, b])		# lead compensator
+Li = Pi*Ci
 
 # Bode plot for the open loop process
-figure(1); 
-bode(Pi);
+plt.figure(1) 
+bode(Pi)
 
 # Bode plot for the loop transfer function, with margins
-figure(2); 
-bode(Li);
+plt.figure(2)
+bode(Li)
 
 # Compute out the gain and phase margins
 #! Not implemented
-# (gm, pm, wcg, wcp) = margin(Li);
+# gm, pm, wcg, wcp = margin(Li)
 
 # Compute the sensitivity and complementary sensitivity functions
-Si = feedback(1, Li);
-Ti = Li * Si;
+Si = feedback(1, Li)
+Ti = Li * Si
 
 # Check to make sure that the specification is met
-figure(3);  gangof4(Pi, Ci);
+plt.figure(3)
+gangof4(Pi, Ci)
 
 # Compute out the actual transfer function from u1 to v1 (see L8.2 notes)
-# Hi = Ci*(1-m*g*Pi)/(1+Ci*Pi);
-Hi = parallel(feedback(Ci, Pi), -m*g*feedback(Ci*Pi, 1));
+# Hi = Ci*(1-m*g*Pi)/(1+Ci*Pi)
+Hi = parallel(feedback(Ci, Pi), -m*g*feedback(Ci*Pi, 1))
 
-figure(4); clf; subplot(221);
-bode(Hi);
+plt.figure(4)
+plt.clf()
+plt.subplot(221)
+bode(Hi)
 
 # Now design the lateral control system
-a = 0.02; b = 5; K = 2;
-Co = -K*tf([1, 0.3], [1, 10]);		# another lead compensator
-Lo = -m*g*Po*Co;
+a, b, K = 0.02, 5, 2
+Co = -K*tf([1, 0.3], [1, 10])  # another lead compensator
+Lo = -m*g*Po*Co
 
-figure(5); 
-bode(Lo);                       # margin(Lo)
+plt.figure(5)
+bode(Lo)  # margin(Lo)
 
 # Finally compute the real outer-loop loop gain + responses
-L = Co*Hi*Po;
-S = feedback(1, L);
-T = feedback(L, 1);
+L = Co*Hi*Po
+S = feedback(1, L)
+T = feedback(L, 1)
 
 # Compute stability margins
-(gm, pm, wgc, wpc) = margin(L);
+gm, pm, wgc, wpc = margin(L)
 print("Gain margin: %g at %g" % (gm, wgc))
 print("Phase margin: %g at %g" % (pm, wpc))
 
-figure(6); clf; 
-bode(L, logspace(-4, 3));
+plt.figure(6)
+plt.clf()
+bode(L, np.logspace(-4, 3))
 
 # Add crossover line to the magnitude plot
 #
 # Note: in matplotlib before v2.1, the following code worked:
 #
-#   subplot(211); hold(True);
+#   plt.subplot(211); hold(True);
 #   loglog([1e-4, 1e3], [1, 1], 'k-')
 #
-# In later versions of matplotlib the call to subplot will clear the
+# In later versions of matplotlib the call to plt.subplot will clear the
 # axes and so we have to extract the axes that we want to use by hand.
 # In addition, hold() is deprecated so we no longer require it.
 #
-for ax in gcf().axes:
+for ax in plt.gcf().axes:
     if ax.get_label() == 'control-bode-magnitude':
         break
-ax.semilogx([1e-4, 1e3], 20 * np.log10([1, 1]), 'k-')
+    ax.semilogx([1e-4, 1e3], 20*np.log10([1, 1]), 'k-')
```

Comment:
Should this be indented?  It means that it could get executed multiple times?

Reply:
Oops!  I misunderstood that.  Will revert it back.

Reply:
Looks like this line is still indented and shouldn't be?